### ISSUE TITLE
1.12.2 Server Crash on startup

### ISSUE THREAD

**Issue by chunkedelement** [BuiltBrokenModding/Atomic-Science#65]
Thanks for bringing this mod to 1.12.2 works in client but not on server startup.
MC 1.12.2, Forge 14.23.5.2772, Atomic Science Mod version=1.12.2-3.10b61.jar
Trying to make a server for a pack but when testing server for 1.12.2 server crashes with this error:
[00:20:48] [Server thread/ERROR] [FML]: An error occurred trying to load an EventBusSubscriber com.builtbroken.atomic.content.machines.sensors.thermal.BlockThermalRedstone for modid atomicscience
java.lang.NoClassDefFoundError: net/minecraft/client/util/ITooltipFlag
        at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:1.8.0_191]
        at java.lang.Class.privateGetDeclaredMethods(Unknown Source) ~[?:1.8.0_191]
        at java.lang.Class.privateGetPublicMethods(Unknown Source) ~[?:1.8.0_191]
        at java.lang.Class.privateGetPublicMethods(Unknown Source) ~[?:1.8.0_191]
        at java.lang.Class.getMethods(Unknown Source) ~[?:1.8.0_191]
Crash:
https://pastebin.com/mnr4D2pB
Thanks

**Comment by DarkGuardsman** [comment]
Check to make sure you didn't download the deobf version and check that your forge is updated to latest. If that does work try a different file from the curse downloads.

**Comment by chunkedelement** [comment]
Thanks, I rolled the atomic science version in the pack back to 1.12.3.06b 19 beta and server starts. Version Alpha 61 it gives the above server crash.
Thanks again for the help.

**Comment by DarkGuardsman** [comment]
I think I spotted the issue, originally I thought it was a compile problem. As its not uncommon for our CI server to fail to obfuscate data (hence why we have alpha builds). Then I re-read the crash this morning and it's likely a missing SideOnly(Client) check on a few methods. I did a quick pass and did find 2 that didn't have the annotation and have fixed them. 

The fix should be up shortly when the automated CI server runs at 1pm Est (not sure if time changed with DST)
